.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: python
+python:
+  - "3.8"
+
+sudo: 'required'
+
+services:
+  - 'docker'
+
+before_install:
+  - sudo apt-get update && sudo apt-get -y install shellcheck
+
+install:
+  - pip install pre-commit
+
+script:
+  - 'make lint'
+  - 'make image'
+
+after_success:
+  - if [[ "$TRAVIS_BRANCH" == "master" ]]; then
```

Comment:
I think we need to check that we're not also on a pull request as the branch will be master on the "PR" build in travis.
```
if [[ "$TRAVIS_BRANCH" = "master" ]] && [[ "$TRAVIS_PULL_REQUEST" = "false" ]]; then
```

Also noticed:
```
/home/travis/.travis/functions: eval: line 110: syntax error: unexpected end of file
```
[here](https://travis-ci.com/github/digirati-co-uk/docker-cleanup/builds/198472082#L350) as it tries to run, not sure what's caused that though

Reply:
Good spot, fixed that now so it won't push on PR's.

I believe the syntax error was due to the absence of semicolons at the end of the statements inside the if-statement. Fixed now.